src/components/nav-bar/nav-bar.component.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import NavBarItem from './nav-bar-item/nav-bar-item.component.jsx';
+import OnlineShopButton from './online-shop-button/online-shop-button.component.jsx';
+import './nav-bar.styles.scss';
+
+const NavBar = () => (
+
+    <div className="nav-bar-container">
+        <div className="items">
+            <NavBarItem>Concept</NavBarItem>
+        </div>
+        <div className="items">
```

Comment:
items is too much of a generic name. 
also remember its not an item itself, its just containing one for layout purposes.

Reply:
changed to nav-bar-item-container